tests/Application/config/sylius/1.11/packages/security.yaml

```diff
@@ -0,0 +1,148 @@
+parameters:
+    sylius.security.admin_regex: "^/%sylius_admin.path_name%"
+    sylius.security.api_regex: "^/api"
+    sylius.security.shop_regex: "^/(?!%sylius_admin.path_name%|new-api|api/.*|api$|media/.*)[^/]++"
+    sylius.security.new_api_route: "/new-api"
+    sylius.security.new_api_regex: "^%sylius.security.new_api_route%"
+    sylius.security.new_api_admin_route: "%sylius.security.new_api_route%/admin"
+    sylius.security.new_api_admin_regex: "^%sylius.security.new_api_admin_route%"
+    sylius.security.new_api_shop_route: "%sylius.security.new_api_route%/shop"
+    sylius.security.new_api_shop_regex: "^%sylius.security.new_api_shop_route%"
+
```

Comment:
```suggestion

```
These have been removed in 1.11 of SyliusSkeleton. Btw, i think we should copy this file:
https://github.com/Sylius/PluginSkeleton/blob/master/tests/Application/config/packages/security.yaml

Reply:
This change will make the build fails on Sylius 1.11 action. Probably there is some configuration in the common files that must be different, I don't know... Anyway, in the master branch, this file Is aligned with the right one from PluginSkeleton and this is only a temp "help" for users of this bundle to allow them to use it also in Sylius 1.11 until we release the v2.